### PR TITLE
Added new command 

### PR DESCRIPTION
**Pull request by Allan-Nava**

### CODE REVIEW

File: README.md
Comment (Emanuele94) [inline README.md:1]: top. i forgot about this particular. my bad!

File: cmd/service/dockerfile/service.go
Comment (Emanuele94) [inline cmd/service/dockerfile/service.go:1]: i do not know if we need a map here as a structured output.
i think that if we make gpt create a dockerfile, we need to find a way to make it put it into a file. 

or mabye we could export a json structured output and parse it with a logic into a docker file that we create with a separate function. lemme know what you think.!

Comment (Allan-Nava) [reply cmd/service/dockerfile/service.go:1]: Already done the creation of file

Comment (Emanuele94) [reply cmd/service/dockerfile/service.go:1]: it's in this function CreateDockerFile correct ? What kind of output do you get from the command ? can you show me ?

**Review by Emanuele94 — APPROVED** [review]

**Pull request merged**